src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue

```diff
@@ -235,6 +243,11 @@ function addWrongAnswer() {
         >
           add question
         </b-button>
+        <a> Game Time (in seconds): </a>
+        <b-form-input
+          id="TimeInput"
+          v-model="configuration.time"
+        ></b-form-input>
```

Comment:
```suggestion
        <b-form-group
        label-cols-lg="6"
        label="Game Time (in seconds)"
        label-for="input-sm"
      >
        <b-form-input
          id="time-input"
          type="number"
          v-model="configuration.time"
        />
      </b-form-group>
```
Make input type number and should show text and input text box in same line

Reply:
applied